common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -37,6 +38,11 @@
   %endif
 </%def>
 
+## A language-direction indicator, suitable for use in class="" attributes.
```

Comment:
Please include a comment explaining why this is a function and how it has to be used (basically I'd like to avoid triggering this bug again)